code/src/components/MovieList.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export const MovieList = () => {
+  const MOVIES_URL = 'https://api.themoviedb.org/3/movie/popular?api_key=e644b2b8739cf7b05450ed105d6749a6&language=en-US&page=1'
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    fetch(MOVIES_URL)
+      .then((response) => response.json())
+      .then((json) => {
+        setMovies(json.results)
+        // fetching data throw setMovies from the API by getting an array from json.results about different movies and information about them. 
+      })
+  }, [])
+
+  return (
+    <>
+      <section className="movie-list">
+        {movies.map((movie) => (
+          <div
+            className="movie-section"
+            key={movie.id}>
+            <Link
+              key={movie.id}
```

Comment:
No need for `key` here, we already added it in line 23. It should only be added to the root element (element which wraps all other) of loop